cmd/detectExecuteScan.go

```diff
@@ -132,8 +132,12 @@ func addDetectArgs(args []string, config detectExecuteScanOptions, utils detectU
 		Version: config.Version,
 	}
 
-	_, detectVersionName := versioning.DetermineProjectCoordinates("", config.VersioningModel, coordinates)
-
+	detectVersionName := config.CustomScanVersion
```

Comment:
shouldn't we read from build descriptor as well (as for WhiteSource)?

https://github.com/SAP/jenkins-library/blob/master/cmd/whitesourceExecuteScan.go#L277

Reply:
Let's handle this in a separate PR.